ci/setup_cache.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-if [[ ! -z "${GCP_SERVICE_ACCOUNT_KEY}" ]]; then
+if [[ ! -z "${GCP_SERVICE_ACCOUNT_KEY:0:1}" ]]; then
```

Comment:
Is that offset/length actually necessary?

Reply:
Because this file is `source`d from `ci/do_ci.sh`, adding `set -x` to that script also dumps this secret in log (I did it while I was debugging locally), only checking the first character prevent those accident.